btf_types/src/lib.rs

```diff
@@ -103,6 +103,8 @@ pub struct BrainFuckProgram {
     filename: PathBuf,
     /// List of instructions with location parsed from file.
     instructions: Vec<IntructionPosition>,
+    /// Mapping for brackets.
+    brackets_map: HashMap<usize, usize>,
```

Comment:
While this approach is "fine" is there one which will result in fewer allocations?  If you can think of one, then also think about if it would be more efficient when used by the interpreter VM - either decide to stick with what you have, or rework, as appropriate, but please let me know what you choose.